interfaces/apparmor/backend.go

```diff
@@ -620,7 +622,7 @@ func (b *Backend) RemoveLate(snapName string, rev snap.Revision, typ snap.Type)
 
 	globs := []string{snapConfineProfileName(snapName, rev)}
 	_, removed, errEnsure := osutil.EnsureDirStateGlobs(dirs.SnapAppArmorDir, globs, nil)
-	// XXX: unloadProfiles() does not unload profiles from the kernel, but
+	// XXX: UnloadProfiles() does not unload profiles from the kernel, but
```

Comment:
(nitpick) it's `UnloadProfiles` (uppercase) in the comment but in the line below it's `unloadProfiles()` (lowecase) - why is this?

Reply:
Ah! Good catch! Because first I thought of renaming the method, then changed my mind, but forgot to update the comment. :-)

Fixed now.